src/components/UpdateVotes.jsx

```diff
@@ -0,0 +1,41 @@
+import { patchVotes } from "../../api";
+import {useState, useEffect} from 'react'
+import {useParams} from 'react-router-dom'
+import '../../styling/Vote.css'
+
+export default function UpdateVotes(){
+    const {article_id} = useParams()
+    const [votesCount, setVotesCount] = useState(0)
+    const [err, setErr] = useState(null)
+
+     const handleVotesClick = () =>{
+        setVotesCount(votesCount+1)
+        patchVotes({article_id}).then(()=>{
+            setErr(null)
+        }).catch((err)=>{
+            setError(err)
+        })
+     }
+
+     const handleDeleteVotesClick = () => {
+        setVotesCount(votesCount - 1)
+        patchVotes({article_id}).then(()=>{
+            setErr(null)
+        })
+        .catch((err)=>{
+            setError(err)
+        })
+     }
+
```

Comment:
You've done a great job of updating the votes in state, but it doesn't look like your patch requests are successful, because the votes are not persisting when you refresh the page. 
- Investigate why the patch requests aren't working
- You are updating an `error` state on line 26, but not using it in the page. Could you use this to give a user feedback if their vote is not successful?